data/json/mapgen/parks_recreation/small_storage_units.json

```diff
@@ -0,0 +1,112 @@
+        "r": "t_pavement",
+        "s": "t_sidewalk",
+        "t": "t_floor",
+        "{": "t_floor",
+        "|": "t_wall"
+      },
+      "furniture": {
+        "#": "f_chair",
+        "+": "f_null",
+        "-": "f_null",
+        ".": "f_null",
+        "6": "f_null",
+        "R": "f_rubble",
+        "^": "f_rubble",
+        "_": "f_null",
+        "b": "f_bed",
+        "c": "f_null",
+        "p": "f_null",
+        "r": "f_rubble",
+        "s": "f_null",
+        "t": "f_table",
+        "{": "f_vending_o",
+        "|": "f_null"
+      },
+      "place_items": [
+        { "item": "novels", "x": 2, "y": 17, "chance": 30 },
+        { "item": "novels", "x": 2, "y": 15, "chance": 10 },
+        { "item": "novels", "x": 22, "y": 7, "chance": 30 },
+        { "item": "novels", "x": 22, "y": 5, "chance": 30 },
+        { "item": "tools_earthworking", "x": 1, "y": 17, "chance": 10 },
+        { "item": "tools_earthworking", "x": 4, "y": 12, "chance": 10 },
+        { "item": "tools_earthworking", "x": 19, "y": 5, "chance": 10 },
+        { "item": "camping", "x": 19, "y": 11, "chance": 30 },
+        { "item": "textbooks", "x": [ 3, 4 ], "y": 9, "chance": 10 },
+        { "item": "tools_electronics", "x": [ 20, 21 ], "y": 17, "chance": 20 },
+        { "item": "tools_electronics", "x": 22, "y": 9, "chance": 20 },
+        { "item": "tools_home", "x": 22, "y": 12, "chance": 15 },
+        { "item": "tools_home", "x": 1, "y": 8, "chance": 25 },
+        { "item": "trash", "x": 4, "y": 21, "chance": 25 },
+        { "item": "trash", "x": 18, "y": 20, "chance": 25 },
+        { "item": "trash", "x": 14, "y": 20, "chance": 25 },
+        { "item": "trash", "x": 18, "y": 15, "chance": 25 },
+        { "item": "trash", "x": 8, "y": 14, "chance": 25 },
+        { "item": "trash", "x": 18, "y": 7, "chance": 25 },
+        { "item": "trash", "x": 12, "y": 4, "chance": 25 },
+        { "item": "alcohol", "x": 22, "y": 14, "chance": 25 },
+        { "item": "jackets", "x": 20, "y": 19, "chance": 10 },
+        { "item": "jackets", "x": 20, "y": 12, "chance": 10 },
+        { "item": "jackets", "x": 2, "y": 11, "chance": 30 },
+        { "item": "jackets", "x": 19, "y": 9, "chance": 10 },
+        { "item": "shoes", "x": 20, "y": 21, "chance": 25 },
+        { "item": "shoes", "x": 1, "y": 19, "chance": 25 }
+      ],
+      "add": [
+        { "item": "holybook_bible1", "x": 22, "y": 11, "repeat": 1 }
+      ],
+      "place_monsters": [
+        { "monster": "GROUP_ZOMBIE", "x": 21, "y": 19, "chance": 6, "density": 1 },
+        { "monster": "GROUP_ZOMBIE", "x": 3, "y": 19, "chance": 6, "density": 1 },
+        { "monster": "GROUP_ZOMBIE", "x": 21, "y": 13, "chance": 6, "density": 1 },
+        { "monster": "GROUP_ZOMBIE", "x": 3, "y": 13, "chance": 6, "density": 1 },
+        { "monster": "GROUP_ZOMBIE", "x": 20, "y": 7, "chance": 6, "density": 1 },
+        { "monster": "GROUP_ZOMBIE", "x": 3, "y": 7, "chance": 6, "density": 1 }
```

Comment:
All `"density": 1` have to be removed. This is redundant, as default density is obviously 1.